src/internal/pacherr/retry.go

```diff
@@ -0,0 +1,45 @@
+package pacherr
+
+import (
+	"errors"
+	"net"
+	"time"
+
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+type Retry struct {
```

Comment:
I'm thinking of renaming this `Transient`

Reply:
If you do, can you include `Err` or `Error` in the name like ~every other error we have?

Reply:
Yeah does `TransientError` sound okay?

Reply:
sgtm